sources/common/doc/examples/vcdMaker.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no" ?>
+<!DOCTYPE signals SYSTEM "vcdMaker.dtd">
+<signals>
+
+<vector>
+    <line>^#([[:d:]]+) ([[:graph:]]+) ([[:d:]]+) ([[:d:]]+)( +.*)?</line>
+    <timestamp>1</timestamp>
+    <name>2</name>
+    <value>3</value>
+    <size>4</size>
+</vector>
+
+<real>
+    <line>^#([[:d:]]+) ([[:graph:]]+) ([[:d:][:punct:]]+) f( +.*)?</line>
+    <timestamp>1</timestamp>
+    <name>2</name>
+    <value>3</value>
+</real>
+
+<event>
+    <line>^#([[:d:]]+) ([[:graph:]]+) e( +.*)?</line>
+    <timestamp>1</timestamp>
+    <name>2</name>
+</event>
+
+<event>
```

Comment:
What is that?

Reply:
It means that the name of the signal is built from the second group of the regex from the 'line' tag. It is the simplified version of our target solution now.

Reply:
I meant: what is the purpose of those blocks? `test1` and `test2` below? Those are not proper regexps, they look like some leftovers from testing.